website/content/docs/security/acl/acl-roles.mdx

```diff
@@ -1,12 +1,11 @@
 ---
 layout: docs
-page_title: Roles
+page_title: ACL Roles
 description: >-
-  This topic describes roles within the access control list (ACL) system. A role is a named set of policies and service identities.
-  They enable you to reuse policies by decoupling the policies from the token distributed to team members.
+Roles are a collection of ACL policies that enable service and node identities. Learn how roles allow you to reuse and update policies without requiring unique tokens for each requestor.
```

Comment:
The current copy is misleading (we want each requestor to have a unique token, we just don't want them to need to reissue tokens if they modify the role definition).

```suggestion
Roles are a named collection of ACL policies, service identities, and node identities. Learn how roles allow you to reuse and update access control policies without needing to distribute new tokens to users.
```